src/main/scala/com/tuplejump/snackfs/cassandra/store/ThriftStore.scala

```diff
@@ -1,736 +0,0 @@
-/*
```

Comment:
We don't want to remove the Thrift store. It is the oldest and most tested and in use store. And even if we don't develop it in future, we would need to keep it for backward compatibility for sometime to come.